src/main/java/uk/gov/justice/digital/delius/data/api/Disability.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.justice.digital.delius.data.api;
+
+import lombok.Builder;
+import lombok.Data;
+
+import java.time.LocalDate;
+
+@Data
+@Builder
+public class Disability {
+    private Long disabilityId;
```

Comment:
Why do we need to surface the Id through the API? Is this the PK or something else?

Reply:
Yeah, primary key for a disability - generally clients need something that can be used to distinguish between the disabilities - though are unlikely to use in subsequent API calls until we have update/delete functionality